src/NServiceBus.Core.Tests/ArgumentExceptionTests.cs

```diff
@@ -136,11 +136,12 @@ bool MethodCallSelf(MethodDefinition method)
 
         static void WriteMethod(MethodDefinition method, TextWriter writer)
         {
-            writer.WriteLine("\r\n" + method.DeclaringType.Name + "." + method.Name);
-            var instruction = method.Body.Instructions.FirstOrDefault(x => x.SequencePoint != null);
+            writer.WriteLine($"\r\n{method.DeclaringType.Name}.{method.Name}");
+            var instruction = method.Body.Instructions.FirstOrDefault(x => x.SequencePoint != null && x.SequencePoint.StartLine != 16707566);
```

Comment:
Is it this http://stackoverflow.com/questions/21839237/how-to-limit-0xfeefee-line-numbers-in-pdb-files ?